src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
I'm interested in how you did these measurements. Are they comparative differentials between versions of Lantern with and without this or a profiling of the function itself?
Normally symmetric encryption has some overhead, that's why I'm interested in this aspect.